unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -433,6 +434,7 @@ loop = do
             QuitI {} -> wat
             DeprecateTermI {} -> undefined
             DeprecateTypeI {} -> undefined
+            GistI {} -> wat
```

Comment:
I've noticed we use `wat` a lot here and I'm curious if we should be doing something else. Where is this input description used?